### PR TITLE
Fix Android Navigation Header

### LINKED ISSUE #695: Android NavigationHeader not receiving press
### Details

Currently on Android, when navigating to a `ContentSingle` in the Navigation Header, the `X` close button is not responding to press. Currently I'm unable to exit a content single without using the android native back button.

### Expected Behavior

When I press the close `X` button, that it responds and closes the view.

### Actual Behavior

Currently, the button does not respond.

### Possible Fix

@conrad-vanl mentioned it could be an issue with a more recent version of React Native.

<details><summary>Additional Info</summary>

### Your Environment

- Environment name and version (e.g. Chrome 39, node.js 5.4): Pixel 2 XL
- Operating System and version (desktop or mobile): Android SDK 28

### Steps to Reproduce

Android-UI

### Stack Trace

</details>

### PR DESCRIPTION
**Pull request by burkeshartsis**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes #695 
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
This PR will work with 0 change in functionality for all current uses. In order for Android to work properly, users will have to make two small tweaks to `NavigationHeader` along with update their package.json versions.

**Comment by vinnyjth** [comment]
@burkeshartsis can you look over my changes, and then merge this?

File: packages/apolloschurchapp/package.json [hunk 1]
```diff
@@ -105,13 +105,13 @@
     "react-native-passkit-wallet": "^0.1.4",
     "react-native-safari-view": "2.1.0",
     "react-native-safe-area-view": "0.13.1",
-    "react-native-screens": "^1.0.0-alpha.22",
+    "react-native-screens": "^1.0.0-alpha.11",
```
Comment (burkeshartsis) [inline packages/apolloschurchapp/package.json]: Would this effectively be a roll back? Was there a particular issue we were trying to avoid?

File: packages/apolloschurchapp/src/content-single/ContentSingle.js [hunk 2]
```diff
@@ -25,6 +25,8 @@ class ContentSingle extends PureComponent {
 
   static navigationOptions = {
     header: NavigationHeader,
+    headerTransparent: true,
+    headerMode: 'float',
```
Comment (burkeshartsis) [inline packages/apolloschurchapp/src/content-single/ContentSingle.js]: 💥

File: packages/apolloschurchapp/src/ui/LandingScreen/__snapshots__/LandingScreen.tests.js.snap [hunk 3]
```diff
@@ -259,15 +259,12 @@ exports[`The Onboarding LandingScreen component should pass additional props to
         }
         vertical={false}
       >
-        <View
-          onLayout={[Function]}
-          pointerEvents="box-none"
-          style={
+        <RCTSafeAreaView
```
Comment (burkeshartsis) [inline packages/apolloschurchapp/src/ui/LandingScreen/__snapshots__/LandingScreen.tests.js.snap]: Do we know where this change came from? Seems unrelated? Was it caused by an the update to React Navigation?

File: packages/apolloschurchapp/package.json (hunk 1 above)
Comment (vinnyjth) [reply packages/apolloschurchapp/package.json]: Yah - we have to match the version on react-navigation. We were on this version of a while, it was updated recently as part of a routine package update.

File: packages/apolloschurchapp/src/ui/LandingScreen/__snapshots__/LandingScreen.tests.js.snap (hunk 3 above)
Comment (vinnyjth) [reply packages/apolloschurchapp/src/ui/LandingScreen/__snapshots__/LandingScreen.tests.js.snap]: Yep!
